python_tools/mixxx_to_rekordbox.py

```diff
@@ -22,10 +22,51 @@
 from utils.track_utils import BeatGridInfo
 from utils.track_utils import guess_inizio_sec
 from utils.track_utils import position_frame_to_sec
+import sqlalchemy.exc
 
 
 # 0 star = "0", 1 star = "51", 2 stars = "102", 3 stars = "153", 4 stars = "204", 5 stars = "255"
 RATING_MAPING = {0: 0, 1: 51, 2: 102, 3: 153, 4: 204, 5: 255}
+
+# Rekordbox color palette (hex values)
+REKORDBOX_COLORS = {
+    (255, 0, 0): "0xFF0000",     # Red
+    (255, 165, 0): "0xFFA500",   # Orange
+    (255, 255, 0): "0xFFFF00",   # Yellow
+    (0, 255, 0): "0x00FF00",     # Green
+    (37, 253, 233): "0x25FDE9",  # Aqua
+    (0, 0, 255): "0x0000FF",     # Blue
+    (102, 0, 153): "0x660099",   # Purple
+    (255, 0, 127): "0xFF007F",   # Pink
+}
+
+def rgb_to_rekordbox_color(rgb_value: int | float) -> str:
+    """Convert Mixxx RGB color to Rekordbox hex color code."""
+    if rgb_value is None or pd.isna(rgb_value):
+        return None
+    
+    # Convert to integer if it's a float
+    rgb_value = int(rgb_value)
+    
+    # Extract RGB components
+    r = (rgb_value >> 16) & 0xFF
+    g = (rgb_value >> 8) & 0xFF
+    b = rgb_value & 0xFF
+    
+    # Find closest Rekordbox color
+    min_distance = float('inf')
+    closest_color = None
+    
+    for rb_rgb, rb_hex in REKORDBOX_COLORS.items():
+        distance = ((r - rb_rgb[0]) ** 2 + 
+                   (g - rb_rgb[1]) ** 2 + 
+                   (b - rb_rgb[2]) ** 2)
+        if distance < min_distance:
+            min_distance = distance
+            closest_color = rb_hex
```

Comment:
This stackoverflow answer has a version that is shorter and probably faster: https://stackoverflow.com/a/58656765